### DISCUSSION TITLE
Default translation and country codes

### DISCUSSION THREAD

**Discussion by WoLfulus** [directus/directus#3681] [Feature Requests]
Would be nice to have a "default" translation language, in which we can use it for non translatable terms and/or, or at least fallback to the registered field name once there's a translation in place.

For example, I registered a field "profissao" which is "Profissão" in pt_BR, but as soon as I registered that translation, I was forced to also create a translation to english (I use english on my user). The default inferred "Profissao" would have been better than "fields.table.profissao", but in this case it would have been way better if we had a "Default" translation option when translating a field or anything else. 

Can even be a special key "default" on the country code.

Also, would be nice to also support language as a fallback instead of country codes. for example registering the same translation for `en` would apply for all `en_*` languages.

**Comment by davidsandoz** [comment]
I also miss that feature. I think this relates to that archived issue: https://github.com/directus/v8-archive/issues/1840

But instead of headers, it could be a query parameter `lang` to go along the other existing query parameters of the API. And in the response, instead of having the translation-specific fields in a `translations` attribute, they could be placed at the same level as the other fields.

**Example:**
- English is set as fallback language
- We have a collection of *Articles*, with the fields `id` and `status`, and the translations in English (`en`) and French (`fr`) for the fields `title` and `body`.
- There is an article (`id: 3`) that has the `title` and `body` provided in English, but not in French.

If I request that article in French (`/items/articles/3?lang=fr`), I would still get the English content:
```json
{
  "data": {
    "id": 3,
    "status": "published",
    "title": "English title",
    "body": "English body"
  }
}
```

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
